webapp/utils.py

```diff
@@ -73,3 +73,18 @@ def load_config_files(directory_name: str):
                 merged = {**merged, **json_content}
     print(json.dumps(merged, indent=2))
     return merged
+
+
+def get_time(string_time):
+    return datetime.strptime(string_time, "%H:%M").time()
+
+
+def get_greeting_msg():
+    current_time = datetime.now().time()
+    if get_time("06:00") <= current_time < get_time("12:00"):
+        return "Доброе утро"
+    if get_time("12:00") <= current_time < get_time("18:00"):
+        return "Добрый день"
+    if get_time("18:00") <= current_time < get_time("22:00"):
+        return "Добрый вечер"
+    return "Доброй ночи"
```

Comment:
Здесь хочется выйти на [табличное решение](https://t.me/kispython/4) :)
```suggestion
def get_greeting_msg():
    greetings = {
        get_time("06:00") <= current_time < get_time("12:00"): "Доброе утро",
        get_time("12:00") <= current_time < get_time("18:00"): "Добрый день",
        get_time("18:00") <= current_time < get_time("22:00"): "Добрый вечер",
        get_time("22:00") <= current_time < get_time("06:00"): "Доброй ночи",
    }
    current_time = datetime.now().time()
    return greetings[current_time]
```